inc/layers.php

```diff
@@ -17,16 +17,6 @@ function add_layers_box_to_post_types(){
       return $supported_post_types;
     }
 
-    function get_localization_language($site=""){
```

Comment:
@Huyeng this function is unnecessary, we can re-use odm_language_manager()->get_the_language_by_site()